components/sidebar/channel_filter/channel_filter.tsx

```diff
@@ -2,25 +2,80 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
+import {FormattedMessage} from 'react-intl';
+import classNames from 'classnames';
```

Comment:
Are you adding this library somewhere? I don't see it in the package.json

Reply:
`classnames` is used in a lot of our components. It's just a simple way of adding classes via booleans. It's been around a while AFAIK.

Reply:
Huh, strange. It looks like it's a dependency-of-a-dependency that someone started using at some point, so that's why it's not in the package.json

Reply:
Oh, we should add that into `package.json` then.